cicd/gitops/configuring-sso-for-argo-cd-using-keycloak.adoc

```diff
@@ -0,0 +1,26 @@
+[id="configuring-sso-for-argo-cd-using-keycloak"]
+= Configuring SSO for Argo CD using keycloak
+include::modules/common-attributes.adoc[]
+include::modules/gitops-document-attributes.adoc[]
+:context: configuring-sso-for-argo-cd-using-keycloak
+
+toc::[]
+
+After the {gitops-title} Operator is installed, Argo CD automatically creates a user with `admin` permissions. To manage multiple users, Argo CD allows cluster administrators to configure SSO.
+
+.Prerequisites
+* Red Hat SSO is installed on the cluster.
+
+include::modules/gitops-creating-a-new-client-using-keycloak.adoc[leveloffset=+1]
```

Comment:
I notice a lot of your procedure modules don't have introductions. Mod. Docs. require an introduction and procedure steps for a procedure module. 

I know introductions can be tedious to write, but they can be pretty short. See my suggestions for each module and let me know what you think :)